phases/register.go

```diff
@@ -124,7 +124,7 @@ func Register(token string, progress Progress, inputReader io.Reader) error {
 		return err
 	}
 
-	fmt.Println(pretty.Object(resp))
+	log.Debugf(pretty.Object(resp))
```

Comment:
This method is still not fixed, still possibly will swallow un-exported fields as it reales on JSON format. We should not use this package until it is fixed.

Reply:
it is not the scope of this task